src/components/Home/Hero/GetStarted/GetStartedWithDvc.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import HeroContainer from '../../../HeroContainer'
+import { StaticImage } from 'gatsby-plugin-image'
+import { cn } from '../../../../utils'
+
+import TwoRowsButtonLink from '../../../TwoRowsButton/link'
+
+import DownloadButton from '../../../DownloadButton'
+
+import DvcSlides from '../../LandingHero/DvcSlides'
+
+import * as styles from './styles.module.css'
+
+import { logEvent } from '@dvcorg/gatsby-theme-iterative/src/utils/front/plausible'
+import ShowOnly from '@dvcorg/gatsby-theme-iterative/src/components/ShowOnly'
+import Link from '@dvcorg/gatsby-theme-iterative/src/components/Link'
+
+const logUseCasesEvent = () => {
+  logEvent('Button', { Item: 'how-it-works' })
+}
+
+const logVSCodeEvent = () => {
+  logEvent('Button', { Item: 'vscode-dvc-ext' })
+}
+
+const HeroButtons = () => (
+  <div className={styles.buttonsContainer}>
+    <ShowOnly on="mobile">
+      <Link
+        className={cn(styles.actionButton, styles.getStartedButton)}
+        href="/doc/start"
+      >
+        Get started
+      </Link>
+    </ShowOnly>
+    <ShowOnly on="desktop">
+      <DownloadButton />
+    </ShowOnly>
+    <TwoRowsButtonLink
+      mode="vscode"
+      className={styles.actionButton}
+      title="DVC For VS Code"
+      description="Get VS Code Extension"
+      icon={
+        <img
+          className={styles.actionButtonIcon}
+          src="/img/vscode-alt-icon.svg"
+          alt="Get VS Code Extension"
+        />
+      }
+      onClick={logVSCodeEvent}
+      href="https://marketplace.visualstudio.com/items?itemName=Iterative.dvc"
+    />
+    <TwoRowsButtonLink
+      mode="outline"
+      className={cn(
+        styles.actionButton,
+        'btn-with-focus',
+        'btn-with-focus--white'
+      )}
+      title="Watch video"
+      description="How it works"
+      icon={
+        <img
+          className={styles.actionButtonIcon}
+          src="/img/play-icon.svg"
+          alt="Watch video"
+        />
+      }
+      onClick={logUseCasesEvent}
+      href="#use-cases"
+    />
+  </div>
+)
+
+const GetStartedWithDvc = () => {
+  return (
+    <HeroContainer className="py-10 px-6" id="get-started-dvc">
+      <div className="flex items-center gap-4 md:justify-center">
+        <h1 className={cn('text-2xl font-medium')}>Get Started with</h1>
+        <StaticImage
```

Comment:
Small thing, but I'd like for the DVC and DVCX logos to feel like they're part of the `h1` here. For example, the gap between "Get started with" and the logo should be about the same as a whitespace character, and the baseline of the 'D' and 'C' in the logo should be aligned with the baseline of the "Get started with" section.

I realize this is probably fiddly, so don't worry too much about it, but I think it'll look nicer if we get these visual details honed in. 

![image](https://github.com/iterative/dvc.org/assets/186715/531d9480-5285-4a7e-9850-5cf7b0a40d4e)